SPECIFICATION.md

```diff
@@ -232,13 +236,51 @@ for ($i ::= 0; $i < length($payload); $i += 1) do
 end
 
 $payloadString ::= join("/", $payload);
-$signatureHex ::= ecdsaSign($payloadString);
```

Comment:
Should we call out that the ECDSA signature is against the SHA256 hash of the payload, not of the raw payload string itself?